libraries/botbuilder-core/tests/storageBaseTests.js

```diff
@@ -171,31 +175,16 @@ class StorageBaseTests {
         assert.strictEqual(reloadedStoreItems3.pocoItem.count, 100);
         assert.strictEqual(reloadedStoreItems3.pocoStoreItem.count, 100);
 
-        // Write with empty etag should not work
-        try {
-            const reloadedStoreItems4 = await storage.read(['pocoStoreItem']);
-            const reloadedStoreItem4 = reloadedStoreItems4.pocoStoreItem;
-
-            assert.notStrictEqual(reloadedStoreItem4, null);
-
-            reloadedStoreItem4.eTag = '';
-            const dict2 = { pocoStoreItem: reloadedStoreItem4 };
-
-            await storage.write(dict2);
-
-            assert.fail('Should have thrown exception on write with storeItem because of empty eTag');
-        } catch (err) { }
-
```

Comment:
Note: I removed this test because:

1. If you read it closely, you can see it will always pass. This is my fault, as I wrote the test, originally.
2. This test merges many *Storage tests into one, ensuring they all pass. As it turns out, [the old tests](https://github.com/microsoft/botbuilder-js/blob/4c01e2fd7bd8febba8a40028bc78dd862d50ec87/libraries/botbuilder-core/tests/memoryStorage.test.js) didn't have a test for this and [would've failed anyway](https://github.com/microsoft/botbuilder-js/blob/main/libraries/botbuilder-core/src/memoryStorage.ts#L80). I believe the test in this file (that this PR removed) actually comes from one I found in .NET, orginally.